QuadradiusR/scripts/context.gd

```diff
@@ -5,6 +5,7 @@ var user_id: String
 var lobby_data: Dictionary
 var game_data: Dictionary
 var game_state: GameState
+var powerups_data: Dictionary
```

Comment:
this will hold all the definitions of powerups so that it wont be necessary to get all the definitions individually